docs/UserGuide.md

```diff
@@ -103,18 +119,73 @@ Beyond individual products, you can also:
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds.
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
+Ta-da! *MyInsuRec* is now installed on your system.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## 4. Quick start
+
+### 4.1 Basic features
+
+Are you new to *MyInsuRec*? This section is the place to start! 
+
+Here, we will be guiding you on the basic features of *MyInsuRec* and help you get familiarized with it.
+
+1. When we first launch *MyInsuRec*, *MyInsuRec* will have already been preloaded with some sample data. We will be using this sample data to get familiar with *MyInsuRec*!
+
+2. Type in `addClient n/Tommy Tan p/81234567` to add a client named 'Tommy Tan' and has a contact number '81234567'. We will see the list of clients update to include this newly added client, with his phone number! The image below shows *MyInsuRec* after adding Tommy.
+
+![AddClientTommy](images/quick-start/AddClientTommy.png)
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:**
+Both client name and their phone number are needed to add the client into *MyInsuRec*.
```

Comment:
Perhaps the word `needed` here can be bold to emphasise the requirements. Also, can consider adding a link here to `addClient` command so that the user can click on it to view the command if necessary.

Reply:
good note on the bolding,

for the addClient command i didn't explicitly link it in the Note bc i had another section with that link, but i think i can add it into the words in the instruction like, is that better?
<img width="314" alt="image" src="https://user-images.githubusercontent.com/63795894/198508373-6513e65f-1a35-4677-a942-388fea3b92c7.png">
'add a client'

Reply:
Yea, I saw the other section where you show the link, just thinking it would be convenient to show it here as well. But I think its fine if we don't include it in here.